package.json

```diff
@@ -50,10 +50,10 @@
     "build": "grunt",
     "test": "grunt test",
     "test-fast": "grunt test-fast",
-    "version": "node build/sri-update",
+    "version": "echo \"use 'npm run release' to bump axe-core version\" && exit 1",
```

Comment:
Why not alias this to `release` or just remove it entirely?

Reply:
Because both `npm version` and `npm run release` bump the version, so if I alliase it its bumped twice, and if I dont put in the error we might accidentally use the wrong command and skip generating the changelog.

Reply:
I like this approach, it makes it clear that `npm version` isn't the way to go.